front/app/components/PostShowComponents/OfficialFeedback/OfficialFeedbackFeed.tsx

```diff
@@ -82,127 +77,123 @@ const LoadMoreButton = styled(Button)`
-        officialFeedbacksList.data &&
-        officialFeedbacksList.data.length > 0
-      ) {
-        const updateDate =
-          officialFeedbacksList.data[0].attributes.updated_at ||
-          officialFeedbacksList.data[0].attributes.created_at;
-        const formattedDate = (
-          <FormattedDate
-            value={updateDate}
-            year="numeric"
-            month="long"
-            day="numeric"
-          />
-        );
-
-        return (
-          <Container
-            aria-live="polite"
-            className={`${className} ${editingAllowed ? 'hasTopMargin' : ''}`}
-          >
-            <FeedbackHeader>
-              <FeedbackTitle>
-                <FormattedMessage {...messages.officialUpdates} />
-              </FeedbackTitle>
-              <FeedbackSubtitle>
-                <FormattedMessage
-                  {...messages.lastUpdate}
-                  values={{
-                    lastUpdateDate: <StyledSpan>{formattedDate}</StyledSpan>,
-                  }}
-                />
-              </FeedbackSubtitle>
-            </FeedbackHeader>
-
-            {officialFeedbacksList.data.map((officialFeedbackPost, i) => {
-              return (
-                <StyledOfficialFeedbackPost
-                  key={officialFeedbackPost.id}
-                  editingAllowed={editingAllowed}
-                  officialFeedbackPost={officialFeedbackPost}
-                  postType="initiative"
-                  a11y_pronounceLatestOfficialFeedbackPost={
-                    i === 0 && a11y_pronounceLatestOfficialFeedbackPost
-                  }
-                />
-              );
-            })}
-
-            {!querying && hasMore && (
-              <LoadMoreButton
-                buttonStyle="secondary-outlined"
-                icon="refresh"
-                onClick={onLoadMore}
-                text={<FormattedMessage {...messages.showPreviousUpdates} />}
-                processing={loadingMore}
-              />
-            )}
-          </Container>
-        );
+const OfficialFeedbackFeed = ({
+  postId,
+  postType,
+  editingAllowed,
+  className,
+  a11y_pronounceLatestOfficialFeedbackPost,
+}: Props) => {
+  const [pageSize, setPageSize] = React.useState(1);
+  const {
+    data: ideaFeedbacks,
+    fetchNextPage: fetchNextPageIdeasFeedback,
+    hasNextPage: hasNextPageIdeasFeedback,
+    isFetchingNextPage: isFetchingNextPageIdeasFeedback,
+  } = useIdeaOfficialFeedback({
+    ideaId: postId && postType === 'idea' ? postId : undefined,
+    pageSize,
+  });
+
+  const {
+    data: initiativeFeedbacks,
+    fetchNextPage: fetchNextPageInitiativeFeedback,
+    hasNextPage: hasNextPageInitiativesFeedback,
+    isFetchingNextPage: isFetchingNextPageInitiativesFeedback,
+  } = useInitiativeOfficialFeedback({
+    initiativeId: postId && postType === 'initiative' ? postId : undefined,
+    pageSize,
+  });
+
+  const officialFeedbacksList =
```

Comment:
Could be memoized since it's doing a bit of work, but might be overkill as well

Reply:
You are right, there is some work involved here but it's pretty minimal. I don't think it's worth memoizing in this case, tbh..